PollyDemos/Demo08_Pipeline-Fallback-WaitAndRetry-CircuitBreaker.cs

```diff
@@ -7,17 +7,31 @@
 namespace PollyDemos;
 
 /// <summary>
-/// Demonstrates using a Retry, a CircuitBreaker and two Fallback strategies.
-/// Same as Demo07 but now uses Fallback strategies to provide substitute values, when the call still fails overall.
-///
-/// Loops through a series of HTTP requests, keeping track of each requested
-/// item and reporting server failures when encountering exceptions.
-///
-/// Observations:
-/// - operation is identical to Demo06 and Demo07
-/// - except fallback strategies provide nice substitute messages, if still fails overall
-/// - OnFallback delegate captures the stats that were captured in try/catches in demos 06 and 07
-/// - also demonstrates how you can use the same kind of strategy (Fallback in this case) twice (or more) in a pipeline.
+/// <para>
+///     Demonstrates using a Retry, a CircuitBreaker and two Fallback strategies. <br/>
+///     Same as Demo07 but now uses Fallback strategies to provide substitute values, when the call still fails overall.
+/// </para>
+/// <para>
+///     Loops through a series of HTTP requests, keeping track of each requested <br/>
+///     item and reporting server failures when encountering exceptions.
+/// </para>
+/// <para>
+///     Observations:
+///     <list type="bullet">
+///         <item>operation is identical to Demo06 and Demo07</item>
+///         <item>except fallback strategies provide nice substitute messages, </item>
+///         <item>if still fails overall OnFallback delegate captures the stats <br/>that were captured in try/catches in demos 06 and 07</item>
```

Comment:
I'm not quite sure what I've added is what you mean, but I didn't understand this item's sentence by itself.

```suggestion
///         <item>if it still fails overall the OnFallback delegate captures the stats <br/>that were captured in try/catches in demos 06 and 07</item>
```

Reply:
I did not change what Ryan wrote. I've just reformatted ir.

Reply:
Before it was two points:

```
- except fallback strategies provide nice substitute messages, if still fails overall
- OnFallback delegate captures the stats that were captured in try/catches in demos 06 and 07
```

Now it's this:

```
- except fallback strategies provide nice substitute messages, 
- if still fails overall OnFallback delegate captures the stats <br/>that were captured in try/catches in demos 06 and 07
```

You've moved text between two list items and now it doesn't read properly.

Reply:
Ohh sh*t... pardon my french :D